subsidy/views.py

```diff
@@ -17,15 +17,26 @@ def default(self, obj):
 
 
 def subsidy_create(request):
+    if request.user.is_anonymous:
+        form= CreateNewSubsidy()
+    else:
+        form = CreateNewSubsidy(initial={'ong': request.user.ong})
     if request.method == "POST":
         form = CreateNewSubsidy(request.POST)
+
         if form.is_valid():
-            form.save()
-            return redirect("/subsidy/list")
+            ong=request.user.ong
+            subsidy=form.save()
```

Comment:
If you do what I said, this line should be changed to `subsidy = form.save(commit=False)`

Reply:
Perfect!

Reply:
You didn't change this either 😢